src/lib.rs

```diff
@@ -1,14 +1,35 @@
-mod core;
+mod asset;
+mod camera;
+#[cfg(feature = "dev")]
+mod dev;
 mod game;
 mod screen;
 mod util;
+mod window;
 
 use bevy::prelude::*;
 
 pub struct AppPlugin;
 
 impl Plugin for AppPlugin {
     fn build(&self, app: &mut App) {
-        app.add_plugins((core::plugin, game::plugin, screen::plugin, util::plugin));
+        // Add Bevy plugins.
+        app.add_plugins(
+            DefaultPlugins
+                .build()
+                // Add `AssetPlugin` via `asset::plugin`.
+                .disable::<AssetPlugin>()
+                .add_after::<AssetPlugin, _>(asset::plugin)
+                // Add `WindowPlugin` via `window::plugin`.
+                .disable::<WindowPlugin>()
+                .add_after::<WindowPlugin, _>(window::plugin),
```

Comment:
flatten those two plugins to here

Reply:
Okay. It's another thing we may want to re-add in the future, but that's fine.